### PR TITLE
refactor, and fix multi monitor problems

### PR DESCRIPTION
**Pull request by henryzx**
#9 fixed

### CODE REVIEW

File: DeveloperExcuses/DeveloperExcusesView.swift [hunk 1]
```diff
@@ -71,54 +70,76 @@ class DeveloperExcusesView: ScreenSaverView {
...
+    
+    func start(){
+        if (!started) {
+            scheduleNext()
+        }
+        started = true;
+    }
     
     func scheduleNext() {
+        NSLog("scheduleNext")
```
Comment (ChristianBecker) [inline DeveloperExcuses/DeveloperExcusesView.swift]: Remove this debug statements?

File: DeveloperExcuses/DeveloperExcusesView.swift [hunk 2]
```diff
@@ -71,54 +70,76 @@ class DeveloperExcusesView: ScreenSaverView {
...
-        }
-        fetchingDue = false
-        
+        NSLog("fetchNext")
         fetchQueue.async { [weak self] in
             guard let data = try? Data(contentsOf: .websiteUrl), let string = String(data: data, encoding: .utf8) else {
                 return
             }
-
+
```
Comment (ChristianBecker) [inline DeveloperExcuses/DeveloperExcusesView.swift]: Revert this lines?

**Comment by clayreimann** [comment]
Is this address by your recent refactor @kimar?

**Comment by kimar** [comment]
Not per se, but this commit should address certain race conditions https://github.com/kimar/DeveloperExcuses/commit/429f79b38f6c591f68d98953df5eaeefe6a46cdb will need to verify the multimonitor issue though

**Comment by kimar** [comment]
Fixed in https://github.com/kimar/DeveloperExcuses/commit/429f79b38f6c591f68d98953df5eaeefe6a46cdb

**Pull request closed without merging**